cli/command/serve/serve.go

```diff
@@ -0,0 +1,20 @@
+package serve
+
+import (
+	"github.com/docker/cli/backends"
+	"github.com/spf13/cobra"
+)
+
+// NewServeCommand command to serve gRPC API for backend commands (ACI/ECS for now) delegated to backend
+func NewServeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                "serve",
+		Short:              "Start a Docker client api server",
```

Comment:
This text sounds like starting Docker REST API server. WDYT about adding the godoc comment above here?